server/src/db/migrations/20230803174046_addUserName.cjs

```diff
@@ -0,0 +1,23 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.table("users", (table) => {
+        table.string("userName").unique().notNullable()
+        table.string("name").notNullable()
```

Comment:
this is a separate feature that in the future should be its own branch to update the user table